CRM/Upgrade/Incremental/php/FiveSeventy.php

```diff
@@ -28,7 +28,20 @@ class CRM_Upgrade_Incremental_php_FiveSeventy extends CRM_Upgrade_Incremental_Ba
    *   The version number matching this function name
    */
   public function upgrade_5_70_alpha1($rev): void {
+    $this->addTask('Add `name` to `civicrm_uf_group', 'addColumn', 'civicrm_uf_group', 'name',
+      "varchar(64) NULL COMMENT 'Form name.'"
+    );
+    // It is important the line above run before the sql, which will populate the fields
+    // before they are made required.
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+
+    $this->addTask('Make civicrm_uf_group.name required', 'alterColumn', 'civicrm_uf_group', 'name',
+      "varchar(255) NOT NULL  COMMENT 'Form name."
```

Comment:
In the schema xml the length of the field is 64 but this says 255.

Reply:
that's because of copy & paste - it's a feature